models/headers-1.0.0.yaml

```
@@ -0,0 +1,475 @@
+# Standard request headers
+
+Accept-Encoding:
+  name: Accept-Encoding
+  in: header
+  required: false
+  description: |-
+    The **Accept-Encoding** indicates the content encoding (usually a
+    compression algorithm) the client understands. The server selects one of
+    the proposed encodings and returns his choice in the **Content-Encoding**
+    response header.
+
+    Supported compression algorithms of the server are [**gzip**][gzip] and
+    **identity**, but other encodings from list below may be support too:
+
+    * **gzip** - a compression format that uses [Lempel-Ziv coding][gzip]
+      (LZ77) with a 32-bit CRC.
+    * **compress** - a format using the [Lempel-Ziv-Welch][lzw] (LZW)
+      algorithm.
+    * **deflate** - A compression format that uses the [zlib][zlib] structure
+      with the [deflate][deflate] compression algorithm.
+    * **br** - a compression format that uses the [Brotli][brotli] algorithm.
+    * **identity** - the identity function without modification or compression
+      used to indicate unchanged responses. This value is always considered as
+      acceptable, even if omitted.
```

```suggestion
    * **gzip** - a[compression format with the [gzip file format][gzip]
        (including a 32 bit CRC)  around the [deflate][deflate] compression algorithm.
    * **deflate** - A compression format that uses the [zlib][zlib] structure
      with the [deflate][deflate] compression algorithm.
    * **compress** - a format using the [Lempel-Ziv-Welch][lzw] (LZW)
      algorithm.
    * **br** - a compression format that uses the [Brotli][brotli] algorithm.
    * **identity** - the identity function without modification or compression
      used to indicate unchanged responses. This value is always considered as
      acceptable, even if omitted.
```
Both gzip and "deflate" (aka zlib) are using the same algorithm (deflate), just with different wrappers.